src/commands/wizard.ts

```diff
@@ -133,9 +138,7 @@ export default class Wizard extends Command {
 
         if (!flags.skipPull) {
             const service = await new BootstrapService();
-            console.log();
-            console.log('Pulling catapult tools image before asking to go offline...');
-            console.log();
+            console.log('\nPulling catapult tools image before asking to go offline...\n');
```

Comment:
I think the idea was to make that particular message more visible. So it's just a refactor, instead of using `console.log()` before and after, used new lines.